data/mods/Xedra_Evolved/design_doc_spoilers.md

```diff
@@ -0,0 +1,175 @@
+
+- Autumn: Fear, decay, the harvest, preservation, fog, colors, home and the hearth
+- Spring: Plants, growth, healing, rain, creativity, and love
+- Summer: Heat, light, athleticism, passionate/strong emotions, illusion, plenty
+- Winter: Darkness, silence, sleep, stasis, ice, snow, cold, wind, death
+
+Seasonal magick requires mana, but it also requires dreamdross to research and the higher-level glamours require dreamdross (refined into "dreamsparks") to weave. This is to provide some resource scarcity, since changelings will have a very large mana pool, and also reinforce that they require access to human dreams to weave their most powerful glamours. No glamour should cost more than 5 dreamsparks to weave, since the absolute maximum capacity is 50 dreamsparks. 
+
+Principles of seasonal magick:
+- Seasonal magick can fail. The chance is highest in opposing seasons (in winter if a summer glamour etc), and is increased by the glamour's Difficulty. It is reduced by deduction skill and by the total number of glamours of that season the changeling knows.
+- Seasonal magic does not have levels. A glamour's power is determined by the changeling's deduction skill and the number of glamours of that season the changeling knows, like with the failure chance.
+- Direct damage should be very rare and usually the result of some natural process caused by the seasonal magic (calling a lightning bolt out of a thunderstorm, starting a wildfire, etc). Folklore has the Fair Folk cursing people and turning them into animals but rarely blowing them up and leaving nothing but smoking boots behind.
+- Seasonal magick needs to be added to the appropriate research system EoC (\Xedra_Evolved\mutations\playable_changeling_seasonal_magic_research_eocs.json) in order to allow it to be discovered. There is one for each season. 
+
+### Iron Vulnerability
+
+The Fair Folk (except Homullus) are all vulnerable to iron, suffering pain when they wear or wield items made of it. This can be merely problematic up to actively crippling, depending on the particular Fair Folk. This is intended to be a core balancing point of the Fair Folk and should not be easily worked around or suppressed.
+
+## Lilin
+
+Lilin (singular lilit) are based on the lilin from Hebrew folklore, plague spirits who haunted the wilderness and abandoned buildings. Like vampires, lilin need to feed on others to survive, but unlike vampires they feed in a more metaphysical sense. They require their target's rauch (Heb רוח, "breath" or "spirit") to survive. This is much more subtle than a vampire's feeding because it requires only touch, or for more powerful lilin, merely close proximity. Being drained of ruach leaves the target listless and enervated, and can eventually kill them. Unlike vampires, lilin can feed on a wide variety of targets, though humans and human-adjacent beings provide the most ruach.
```

Comment:
```suggestion
Lilin (singular lilit) are based on the lilin from Hebrew folklore, plague spirits who haunted the wilderness and abandoned buildings. Like vampires, lilin need to feed on others to survive, but unlike vampires they feed in a more metaphysical sense. They require their target's ruach (Heb רוח, "breath" or "spirit") to survive. This is much more subtle than a vampire's feeding because it requires only touch, or for more powerful lilin, merely close proximity. Being drained of ruach leaves the target listless and enervated, and can eventually kill them. Unlike vampires, lilin can feed on a wide variety of targets, though humans and human-adjacent beings provide the most ruach.
```